src/engine.cpp

```diff
@@ -96,32 +99,14 @@ void Engine::run()
     {
         _keyboard->setKeyBinding(itr->name.GetString(), itr->value.GetInt());
     }
-    // BEGIN TESTCODE
 
-    /*
-    _keyboard->setKeyBinding("left", 263);
-    _keyboard->setKeyBinding("down", 264);
-    _keyboard->setKeyBinding("up", 265);
-    _keyboard->setKeyBinding("right", 262);
-    //*/
-
-    std::string shaderN = loadShader("name", "vert.glsl", "frag.glsl");
-
-    loadTexture("remyd", "image.png");
-
-    GameObject* child = world.addChild("remyd");
-    components::RenderComponent* childRenderer =
-        new components::RenderComponent(child);
-
-    child->addComponent("renderer", childRenderer);
-
-    childRenderer->texture = "remyd";
-    childRenderer->shader = shaderN;
-    childRenderer->tint = graphics::Colors::Red;
-
-    child->transform.scale = math::Vector3<float>(64.0f, 64.0f, 1.0f);
-
-    // END TESTCODE
+#ifdef NAUTICAL_BIND_PYTHON
```

Comment:
That's done by moving the #endif here to right after the addLoader call - everything else will work regardless of what script type it is.